package.json

```diff
@@ -3,11 +3,15 @@
   "version": "10.8.0-beta.1",
   "description": "Format whitespace in a SQL query to make it more readable",
   "license": "MIT",
-  "main": "dist/sql-formatter.min.js",
+  "main": "dist/sql-formatter.min.cjs",
   "module": "lib/index.js",
   "types": "lib/src/index.d.ts",
   "exports": {
-    ".": "./lib/index.js"
+    ".": {
+      "import": "./lib/index.js",
+      "require": "./dist/sql-formatter.min.cjs",
+      "types": "./lib/src/index.d.ts"
```

Comment:
> the "types" condition should always come first in "exports"

See https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/#package-json-exports-imports-and-self-referencing

Reply:
Thanks a lot. Wouldn't have guessed this gotcha on my own.